program_map.go

```diff
@@ -1,54 +1,41 @@
 package astits
 
-import "sync"
-
 // programMap represents a program ids map
 type programMap struct {
-	m *sync.RWMutex
-	p map[uint16]uint16 // map[ProgramMapID]ProgramNumber
+	p map[uint32]uint16 // map[ProgramMapID]ProgramNumber
```

Comment:
Can you add the following comment on top of `p` : `// We use map[uint32] instead map[uint16] as go runtime provide optimized hash functions for (u)int32/64 keys`

Reply:
Done